src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
\* _wince_ *

I know what you are doing here because I've had to do it at work.

Is this really necessary? You have much more control over this code base and much less code. You can apply more intelligent patterns that would make this unnecessary.

Mockito is powerful, but there are problems that are better suited to applying intelligent patterns to instead.